Add missing derivatives.yaml input

zdevito opened the pull request.

zdevito committed:
Add missing derivatives.yaml input

zdevito commented:
#4079

colesbury commented on tools/setup_helpers/generate_code.py:
```diff
@@ -17,7 +17,8 @@ def all_generator_source():
     'torch/csrc/generic/TensorMethods.cwrap',
     'torch/lib/tmp_install/share/ATen/Declarations.yaml',
-    'torch/lib/tmp_install/share/ATen/Declarations.yaml'
+    'torch/lib/tmp_install/share/ATen/Declarations.yaml',
```
Why is this listed twice?

colesbury reviewed (commented):
Should this include all the files in the directory torch/csrc/generic/methods?

zdevito commented:
I added them. Hopefully that is all the deps. I added some logic to the aten build last week to _force_ new dependencies to be explicitly reflected in the build, but I am not sure it is worth doing for all of the generate code in pytorch.

colesbury reviewed (approved):
lgtm

ezyang commented:
Thanks!
